src/routes/api/[...endpoint]/proxy.ts

```diff
@@ -11,6 +11,10 @@ async function _fetchApi(
   const url = new URL(relativeUrl, PUBLIC_API_URL);
   headers ??= new Headers();
   headers.append('Content-Type', 'application/json');
+  headers.append('Accept', 'application/json');
+  if (window?.location?.origin) {
+    headers.append('Origin', window.location.origin);
+  }
```

Comment:
I think the `Origin` header is automatically added by browsers when `fetch` is called, so it shouldn't be necessary.

Reply:
It is, it's just a way to make sure it is there. I'm not sure how could we have a cors problem otherwise.

Reply:
I think the cors problem was because `OPTIONS` requests (used by CORS) weren't being proxied to the backend. I'll approve either way.